### DISCUSSION TITLE
Is the code that was used for training available?

### DISCUSSION THREAD

**Discussion by panademo** [openai/whisper#44] [Q&A] [answered]
Hi,  incredible model, thank you for having released this as OSS, and my apologies if this is a dumb question, but has the training code been made available/open-sourced as well?

**Comment by jongwook** [comment, marked as answer]
We currently don't have plans to release training/fine-tuning code, but there might be an implementation from the community soon.

**Comment by sanchit-gandhi** [comment]
And here it is! Check-out this blog for fine-tuning Whisper for multilingual ASR with Hugging Face Transformers: https://huggingface.co/blog/fine-tune-whisper

It provides a step-by-step guide to fine-tuning, right from data preparation to evaluation 🤗 There'a Google Colab so you can also run it as a notebook 😉
